README.md

```diff
@@ -156,7 +157,6 @@ QR code
 ```
 
 [^ico-note]: CUR is also supported.
-
```

Comment:
Don't remove this blank line.

<https://prettier.io/blog/2024/06/01/3.3.0#add-newline-between-markdown-footnote-definitions-16063httpsgithubcomprettierprettierpull16063-by-atemahttpsgithubcomatema>